exchanges/order/orders.go

```diff
@@ -741,6 +718,57 @@ func (t Type) String() string {
 	}
 }
 
+// String implements the stringer interface.
+func (t TimeInForce) String() string {
```

Comment:
Could we move all the TimeInForce to one section of code please? Move all of this lower to where other TIF functions are?